share/hooks/vcenter/create_vcenter_net.rb

```diff
@@ -208,9 +255,9 @@ def update_net(vnet, content)
         # We must update XML so the VCENTER_NET_REF and VCENTER_INSTANCE_ID are added
 
         if blocked
-            update_net(one_vnet,"VCENTER_NET_REF=\"#{vnet_ref}\"\nVCENTER_INSTANCE_ID=\"#{vc_uuid}\"\nVCENTER_NET_STATE=\"ERROR\"\nVCENTER_NET_ERROR=\"vnet already exist in vcenter\"\n")
+            update_net(one_vnet,"NSX_ID=\"#{logical_switch.ls_id}\"\nNSX_VNI=\"#{ls_vni}\"\nVCENTER_NET_REF=\"#{vnet_ref}\"\nVCENTER_INSTANCE_ID=\"#{vc_uuid}\"\nVCENTER_NET_STATE=\"ERROR\"\nVCENTER_NET_ERROR=\"vnet already exist in vcenter\"\n")
```

Comment:
i think it would be better to construct the string beforehand in a multiline statement

Reply:
does rubocop apply to this hook?

Reply:
No, rubocop has been only applied to new files. I'm going to apply it to the hooks too.